docs/governance-and-automation/automated-workflows/workspace-type-and-reviewers.md

```diff
@@ -0,0 +1,47 @@
+---
+
+description: This article explains which workspace types can have which policies applied and who can be selected as a reviewer.
+
+---
+
+# Policies: Workspace Types and Reviewers
+
+Before applying the policies you've created or enabled in Syskit Point, it's useful to know which workspaces policies can be applied to and who can be selected as reviewers. 
+
+Below you can find details on:
+
+* **Which types of workspaces** (Microsoft Teams, Microsoft 365 Groups, SharePoint Sites, etc.) are supported by each policy
+
+* **Which users can be assigned as reviewers** to complete tasks created by the policy
+
+
+:::info
+
+**Please note the following**: 
+
+* Policies that are user-centered do not have specific workspaces they can be applied to as they are applied to users in your tenant. 
+* For certain policies with task delegation enabled, when a task is not resolved by workspace owners, you can select secondary reviewers. If workspace owners do not complete the task, it will be reassigned to the secondary reviewers you have chosen.
```

Comment:
nigdje ne koristimo pojam secondary reviewera
koristio bih terminologiju koja se koristi u Pointu
npr.
* For certain policies with task delegation enabled, when workspace owners don't resolve their tasks in time, you can select to whom the task will be reassigned. If workspace owners do not complete the task, it will be reassigned to the reviewers you choose.